src/promptflow-tools/promptflow/tools/common.py

```diff
@@ -350,3 +363,22 @@ def find_referenced_image_set(kwargs: dict):
     except ImportError:
         pass
     return referenced_images
+
+
+def connection_mapping(connection):
```

Comment:
add comment here to share the knowledge why need this conversion.

Reply:
"""
    Normalizes the configuration of a given connection object for compatibility.
    
    This function takes a connection object and normalizes its configuration, 
    ensuring it is compatible and standardized for use.
    """